### PR TITLE
Add a test_model.py script to test a given model on historical data

### PR DESCRIPTION
**Pull request by 4n4nd**
And also log some accuracy metrics to a mlflow server

### CODE REVIEW

File: test_model.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            (
+                ((true_values.metric_values["y"] >= true_values.metric_values["yhat_upper"]))
+                | (true_values.metric_values["y"] <= true_values.metric_values["yhat_lower"])
+            ),
+            1,
+            0,
+        )
+
+        # Label True Anomalies
+        thresold_value = 0.001
```
Comment (MichaelClifford) [inline test_model.py]: I think the `threshold_value` should be a configurable parameter so that user can tune it based on their specific test case.

Comment (MichaelClifford) [inline test_model.py]: Also, having the auto-labeling as its own function might make it clearer for another user to update it in the future, if the current approach needs to be changed. Only pointing it out as the auto-labeling is a somewhat important part of this testing procedure.

Comment (4n4nd) [reply test_model.py]: good idea

Comment (4n4nd) [reply test_model.py]: Changed it, and some parameters are now being logged to mlflow

Comment (4n4nd) [reply test_model.py]: Created a function and adding a comment but I don't know how clear it is

**Pull request merged**